capplets/appearance/appearance-main.c

```diff
@@ -220,9 +228,9 @@ main (int argc, char **argv)
     if (w != NULL) {
       gint pindex;
 
-      pindex = gtk_notebook_page_num (nb, w);
+      pindex = gtk_notebook_page_num (GTK_NOTEBOOK (nb), w);
       if (pindex != -1)
-        gtk_notebook_set_current_page (nb, pindex);
+        gtk_notebook_set_current_page (GTK_NOTEBOOK (nb), pindex);
```

Comment:
if you use `GTK_NOTEBOOK (nb)` more than once, you can assign it to a local variable, for example within the scope of the if block.

Reply:
Like this?